### PR TITLE
osbuilder: Drop Clear Linux

### PR DESCRIPTION
**Pull request by fidencio**
The Clear Linux rootfs is not being tested anywhere, and it seems Intel doesn't have the capacity to review the PRs related to this (combined with the lack of interested from the rest of the community on reviewing PRs that are specific to this untested rootfs).

With this in mind, I'm suggesting we drop Clear Linux support and focus on what we can actually maintain.

### CODE REVIEW

**Comment by fidencio** [comment]
cc @chavafg

**Review by stevenhorsman — APPROVED** [review]
Makes sense to me. I wonder if we should consider a similar exercise for the alpine and debian based rootfs images as I don't think we test them either, but that's out of scope for this PR

**Comment by fidencio** [comment]
> Makes sense to me. I wonder if we should consider a similar exercise for the alpine and debian based rootfs images as I don't think we test them either, but that's out of scope for this PR

I think IBM relies on the alpine rootfs (but, we need to check with @skaegi on that).
The debian one could be removed, unless someone steps up to maintain it.

File: docs/design/architecture/guest-assets.md [hunk 1]
```diff
@@ -135,7 +135,6 @@ See also the [process overview](README.md#process-overview).
 
 | Image type | Default distro | Init daemon | Reason | Notes |
 |-|-|-|-|-|
-| [image](background.md#root-filesystem-image) | [Clear Linux](https://clearlinux.org) (for x86_64 systems)| systemd | Minimal and highly optimized | systemd offers flexibility |
```
Comment (chavafg) [inline docs/design/architecture/guest-assets.md:138]: should we keep the line and just `s/Clear Linux/Ubuntu/` ?
And maybe change the Reason to: `Fully tested in our CI`

**Comment by fidencio** [comment]
I am marking this as a draft till we hear back from @olegs-repo on https://github.com/kata-containers/kata-containers/pull/10436 (or for one week, whatever happens first).

**Comment by fidencio** [comment]
I am merging this one according to: https://github.com/kata-containers/kata-containers/pull/10436#issuecomment-2480760568

**Pull request merged**